airship.properties

```diff
@@ -1,3 +1,3 @@
-crossPlatformVersion = 12.0.1
+crossPlatformVersion = 13.0.0
```

Comment:
did this need to change? it's probably going to conflict with the release branch

Reply:
No need to change this here